lms/djangoapps/courseware/tests/test_date_summary.py

```diff
@@ -42,18 +41,14 @@
 from lms.djangoapps.verify_student.tests.factories import SoftwareSecurePhotoVerificationFactory
 from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
 from openedx.core.djangoapps.content.course_overviews.tests.factories import CourseOverviewFactory
-from openedx.core.djangoapps.self_paced.models import SelfPacedConfiguration
```

Comment:
Just to tag all places where you are removing self_paced functionality please clarify why do we need to remove it here as well?

Reply:
got it thanks